code/modules/mob/living/carbon/superior_animal/robots/subtype/mining/mining_drone.dm

```diff
@@ -20,6 +20,61 @@
 	var/pickup = TRUE // Do we pick up the ore?
 	var/mining = TRUE // Do we mine rocks?
 
+//Putting this here do to no idea were it would fit other then here
+/mob/living/carbon/superior_animal/robot/mining/verb/toggle_viewRange()
+	set name = "Toggle Viewrange"
+	set desc = "Toggles viewrange on the mining bot, smaller ones makes it more easy to mannully controle the bot, well longer range will be more automatic."
+	set category = "Mob verbs"
+
+	if (viewRange == 6)
+		viewRange = 3
+		to_chat(src, SPAN_NOTICE("You toggle the mobs Viewrange to 3."))
+
+	if (viewRange == 3)
+		viewRange = 1
+		to_chat(src, SPAN_NOTICE("You toggle the mobs Viewrange to 1."))
+
+	else
+		viewRange = 6
+		to_chat(src, SPAN_NOTICE("You toggle the mobs Viewrange to 6."))
+
+/mob/living/carbon/superior_animal/robot/mining/verb/toggle_wandering()
+	set name = "Toggle wandering"
+	set desc = "Toggles on/off the mobs wandering."
+	set category = "Mob verbs"
+
+	if (wandering)
```

Comment:
You could do `wandering = !wandering` followed by a `to_chat("You [wandering ? "" : "de"]activate automatic wandering.")`

Reply:
Also why would a controlling ghost want to turn on the wandering? It will just make the bot constantly moving to a random tile.

Reply:
To turn it on when ghosting 
or to turn it to not on when controling